models/bridges/ethereum/bridges_ethereum_addresses.sql

```diff
@@ -0,0 +1,152 @@
+{{ config(alias = 'addresses',
+        tags=['static'],
+           schema = 'bridges_ethereum',
+          post_hook='{{ expose_spells(\'["ethereum"]\',
```

Comment:
can you remove the post-hook, we'll save it to add on the cross-chain view when all blockchains are merged